binary/defaults/pgbinary.defaults

```diff
@@ -357,2474 +357,517 @@
+    History_Track_reverse_yaxis(:) = .false.
+    History_Track_log_xaxis(:) = .false.
+    History_Track_log_yaxis(:) = .false.
 
       ! ::
 
-   show_History_Track3_annotation1 = .false.
-   show_History_Track3_annotation2 = .false.
-   show_History_Track3_annotation3 = .false.
+    History_Track_xmin(:) = -101d0
+    History_Track_xmax(:) = -101d0
+    History_Track_ymin(:) = -101d0
+    History_Track_ymax(:) = -101d0
 
       ! ::
 
-   History_Track3_fname = ''
-
-      ! Enables calling a subroutine to add extra information to a plot
-      ! see ``$MESA_DIR/binary/other/pgbinary_decorator.f90``
-
-      ! ::
-
-   History_Track3_use_decorator = .false.
-
-      ! file output
-
-      ! ::
-
-   History_Track3_file_flag = .false.
-   History_Track3_file_dir = 'png'
-   History_Track3_file_prefix = 'track3_'
-   History_Track3_file_interval = 5
-   History_Track3_file_width = -1
-   History_Track3_file_aspect_ratio = -1
-
-
-      ! History_Track4
-      ! --------------
-
-      ! ::
-
-   History_Track4_win_flag = .false.
-   History_Track4_win_width = 6
-   History_Track4_win_aspect_ratio = 0.75
-
-      ! ::
-
-   History_Track4_xleft = 0.15
-   History_Track4_xright = 0.85
-   History_Track4_ybot = 0.15
-   History_Track4_ytop = 0.85
-   History_Track4_txt_scale = 1.0
-   History_Track4_title = 'History_Track4'
-
-      ! ::
-
-   History_Track4_xname = ''
-   History_Track4_yname = ''
-   History_Track4_xaxis_label = ''
-   History_Track4_yaxis_label = ''
-   History_Track4_reverse_xaxis = .false.
-   History_Track4_reverse_yaxis = .false.
-   History_Track4_log_xaxis = .false.
-   History_Track4_log_yaxis = .false.
-
-      ! ::
-
-   History_Track4_xmin = -101d0
-   History_Track4_xmax = -101d0
-   History_Track4_ymin = -101d0
-   History_Track4_ymax = -101d0
-
-      ! ::
-
-   History_Track4_xmargin = 0.1
-   History_Track4_ymargin = 0.1
-   History_Track4_dxmin = -1
-   History_Track4_dymin = -1
+    History_Track_xmargin(:) = 0.1
+    History_Track_ymargin(:) = 0.1
+    History_Track_dxmin(:) = -1
+    History_Track_dymin(:) = -1
 
       ! ::
 
-   History_Track4_step_min = -1
-   History_Track4_step_max = -1
+    History_Track_step_min(:) = -1
+    History_Track_step_max(:) = -1
 
       ! ::
 
-   show_History_Track4_target_box = .false.
-   History_Track4_n_sigma = -3
-   History_Track4_xtarget = 0
-   History_Track4_xsigma = 0
-   History_Track4_ytarget = 0
-   History_Track4_ysigma = 0
+    show_History_Track_target_box(:) = .false.
+    History_Track_n_sigma(:) = -3
+    History_Track_xtarget(:) = 0
+    History_Track_xsigma(:) = 0
+    History_Track_ytarget(:) = 0
+    History_Track_ysigma(:) = 0
 
       ! ::
 
-   show_History_Track4_annotation1 = .false.
-   show_History_Track4_annotation2 = .false.
-   show_History_Track4_annotation3 = .false.
+    show_History_Track_annotation1(:) = .false.
+    show_History_Track_annotation2(:) = .false.
+    show_History_Track_annotation3(:) = .false.
```

Comment:
Did we want this to become a 2d array (I.e. `show_History_Track_annotation(:,:) = .false.`)?

Reply:
I don't know exactly? Philosophically, these are different, as there aren't `pgstar_array_length` of them, only 3. I'd leave this as is.